charts/aws-ebs-csi-driver/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: "0.8.1"
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
 version: 0.8.2
```

Comment:
the version should also be bumped as technicaslly this is a new helm chart with new YAMLs

It's fine if the helm chart `version` goes out of sync with the driver `appVersion` (it already is)

Reply:
Hi Matthew, should we bump the driver version to 0.9.0?

Reply:
hmm yes that's a good idea, never mind then, let's leave version untouched for now. 

Since there is already demand, we can release driver 0.9.0 + helm chart 0.9.0 later, after this merges.

Reply:
I agree with @wongma7. We should just bump to 0.8.3.

Reply:
Roger, so I will just update chart version to 0.8.3 in this PR and we can sync the driver version and chart version later(in 0.9.0 release)